docsite/docs/api_pages/object-detection.md

```diff
@@ -10,4 +10,424 @@ For more details or to demo it, visit
 [MediaPipe - Object Detection](https://mediapipe-studio.webapps.google.com/studio/demo/object_detector)
 
 
-## How to get started 
+## How to get started
+** Using yarn **
+
+**Requirements**
+* Gradle minimum SDK 24 or higher
+* Android-SDK Version 26 or higher
+* iOS 12 or higher
+
+1. **Open Terminal or Command Prompt:** Open your terminal or command prompt application. Create a brand new React Native Project. In this case we will be using **'rnmpdemo'** as on React Native MediaPipe Demo for example.
+
+```jsx
```

Comment:
should be 
\```bash 

or just 

\```

Reply:
I was using jsx to make Docusaurus "recognize" the language so that way when is up in the site is easier to read since it would be kind of color coded. Is that a bad practice?